### PR TITLE
Fix nightly build for update of RangeInclusive (inclusive_range_methods)

### PR DESCRIPTION
**Pull request by dvic**

### CODE REVIEW

**Comment by dvic** [comment]
Any idea why the build fails? The following is the problem:

```
  --> mockers/tests/multi_trait.rs:17:5
   |
17 |     #[derive_mock(module="::derive_inherited_trait")]
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: message: unexpected name-value attribute param
```

**Comment by kriomant** [comment]
Looking at it…

**Comment by kriomant** [comment]
Change line in lib.rs:
```let attr = syn::parse_outer_attr(&format!("#[mocked{}]", attr_tokens)).expect("parsed");```
to
```let attr = syn::parse_outer_attr(&format!("#[mocked({})]", attr_tokens)).expect("parsed");```
(braces are added).

File: mockers/src/lib.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#![cfg_attr(feature="nightly", feature(fnbox))]
+#![cfg_attr(feature="nightly", feature(fnbox,inclusive_range_methods))]
```
Comment (kriomant) [inline mockers/src/lib.rs]: Space, please.

**Comment by dvic** [comment]
Ah I see. I have fixed your comments and rebased into a single commit. Build is not failing anymore.

**Comment by kriomant** [comment]
I've published version 0.10.2 with your fix. Thank you.

**Pull request merged**